modules/nw-egress-router-network-attachment.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/deploying-egress-router-ovn-redirection.adoc
+
+ifeval::["{context}" == "deploying-egress-router-cni-redirection"]
+:redirect:
+:router-type: redirect
+endif::[]
+
+[id="nw-egress-router-network-attachment_{context}"]
+= Network attachment definition for an egress router in {router-type} mode
+
+Before a pod can act as an egress router, you must specify the network interface configuration as a `NetworkAttachmentDefinition` object.
+The object specifies information such as the IP address to attach to the egress router pod, network destinations, and a network gateway.
+As the pod for the egress router starts, Multus uses the network attachment definition to add a network interface with the specified properties to the pod.
+
+.Example network attachment definition
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: egress-router-redirect  <.>
+spec:
+  config: '{
+    "cniVersion": "0.4.0",
+    "type": "egress-router",
+    "name": "egress-router",
+    "ip": {
+      "addresses": [
+        "192.168.12.99"  <.>
+        ],
+      "destinations": [
+        "192.168.12.91"  <.>
+      ],
+      "gateway": "192.168.12.1"  <.>
+      }
+    }'
+----
+<.> The name of the network attachment definition is used later in the specification for the egress router pod.
+
+<.> The `addresses` key specifies the reserved source IP address to use with the additional network interface.
```

Comment:
It might be worth checking if this array can hold only a single value, or if multiple values are permissible.

Reply:
multiple values *will* be permissible, stay with 1 for 4.7 just in case